examples/serve/demo/config.yml

```diff
@@ -1,6 +1,6 @@
 Authentication:
-  Domain: xcube-dev.eu.auth0.com
-  Audience: https://xcube-dev/api/
+  Domain: kc.brockmann-consult.de/auth/realms/AVL
```

Comment:
I do not agree on the changes of this file. Please revert or explain, why there is everything commented and why you use the authentication dor AVL here. We have a separate configuration file for AVL.